test/stress_tests.py

```diff
@@ -221,6 +223,7 @@ def ray_start_reconstruction(request):
         _internal_config=json.dumps({
             "initial_reconstruction_timeout_milliseconds": 200
         }))
+    ray.worker._init(ray_params)
```

Comment:
We should really deprecate this `ray.worker._init` stuff and instead only use `ray.test.cluster_utils` to simulate multiple nodes on a single machine (certainly not in this PR).